browser-extension/src/rateLaterHistory.js

```diff
@@ -1,5 +1,10 @@
 const RATE_LATER_BULK_MAX_SIZE = 20;
 
+// These selectors can change regularly on YT, and become obsolete.
+// We need to check their validity from time to time.
+const YT_VIDEO_CARD_TAG = 'ytd-item-section-renderer';
```

Comment:
From what I can see, the videos are grouped by date on the history page, and `<ytd-item-section-renderer>` may contain multiples videos that have been watched on the same day. Some the selector needs to be more specific:
 ```suggestion
const YT_VIDEO_CARD_TAG = 'ytd-item-section-renderer yt-lockup-view-model';
```

`yt-lockup-view-model`  alone would not be correct either, as the element is used elsewhere on the same page.

Reply:
Alright, I'll review the selectors.